docs/_toc.yml

```diff
@@ -36,6 +36,10 @@ parts:
     - file: optimization/grid_search
     - file: optimization/feature_ranking
 
+  - caption: Dimensionality Reduction
```

Comment:
We might want to unify this topic under use cases. Thoughts?

Reply:
yes I thought so too, but wasn't sure which one. maybe clustering? or create another topic named `Feature Selection` (If we want to add more functionality with respect to feature selection)

Reply:
Thoughts? @edublancas

Reply:
I think let's merge what we have, later we can think how to re-org the docs if needed.

Reply:
Agree, this has been going on for a while, it should happen soon due to SEO.